src/network/network_gui.cpp

```diff
@@ -326,12 +326,10 @@ class NetworkGameWindow : public Window {
 	void UpdateListPos()
 	{
 		this->list_pos = SLP_INVALID;
```

Comment:
This would only need to be set in the 'not found' return path. This might show that intent more clearly.
```
if (it == std::end) {
    this->list_pos = SLP_INVALID;
} else {
    this->list_post = static...;
}
```
But then... this might also be just bike-shedding, right?